README.md

```diff
@@ -57,16 +73,11 @@ Incubator Charts are those that do not meet these criteria. Having the incubator
 
 In order to get a Chart from incubator to stable, Chart maintainers should open a pull request that moves the chart folder.
 
-## Supported Kubernetes Versions
-
-This chart repository supports the latest and previous minor versions of Kubernetes. For example, if the latest minor release of Kubernetes is 1.8 then 1.7 and 1.8 are supported. Charts may still work on previous versions of Kubernertes even though they are outside the target supported window.
 
```

Comment:
This sentence appears to be out of place now.

Reply:
I believe this is corrected now and is in the same place as before (at the end of that section).